lib/representer/cli.ex

```diff
@@ -6,28 +6,24 @@ defmodule Representer.CLI do
   def main([slug, input_path, output_path]) do
     exercise = slug |> ConvertSlug.kebab_to_snake()
 
-    # save the cwd to go back later
-    cwd = File.cwd!()
+    # input_path should be a path to a mix project, so the solution is somewhere in /lib
+    input_path = Path.join([Path.absname(input_path), "lib"])
+    output_path = Path.absname(output_path)
+    concatenated_input_file_path = Path.join([input_path, @concatenated_file])
 
-    input_path |> Path.absname() |> File.cd!()
-    output_path |> Path.absname() |> File.cd!()
-
-    # get all .ex files, put the exercise file last
+    # get all .ex files
     files =
-      File.ls!()
-      |> Enum.filter(fn f -> String.ends_with?(f, ".ex") end)
+      File.ls!(input_path)
+      |> Enum.filter(fn f -> String.ends_with?(f, ".ex") && f != @concatenated_file end)
       |> Enum.sort_by(fn f -> f == "#{exercise}.ex" end)
```

Comment:
I'm not sure why this sorting would be needed, but it won't work for all concept exercises, e.g. `stack-underflow` uses the file `lib/rpn_calculator/exception.ex`

Reply:
So, like you probably know, the representer serves as a sort of "key generator" for the representer system.  So to produce consistent keys I was sorting the files so that they appear in the concatenated file in the same order